lib/hammer_cli_foreman/api/session_authenticator_wrapper.rb

```diff
@@ -28,13 +29,22 @@ def status
         end
       end
 
+      def force_user_change
+        @user_changed = true
+      end
+
+      def user_changed?
+        @user_changed
```

Comment:
usually `!!@user_changed` is used to ensure the result of method with `?` at the end returns boolean